lib/exponent-server-sdk.rb

```diff
@@ -79,7 +79,12 @@ def handle_success(response)
       end
 
       def extract_data(response)
-        response.fetch('data').first
+        data = response.fetch('data')
+        if data.class == Array
+          data.first
```

Comment:
If the response contains multiple push receipts -- the common case since batched notifications are highly recommended over unbatched ones -- we need to handle each element in the array

Reply:
@ide I'm not certain of the expected behavior expected from Expo. 

My guess is if there's any receipt that doesn't have a status of `ok` then we raise an error. 
I just implemented it in my newest commit.

Although it's not the prettiest, due to having to return response on lines `83` and `87`, I do believe it works.  

Let me know if you guys would like something else.

Lastly, awesome work on Expo! Use it every day and can't say thank you enough.